File: C#-part-1/06.Loops/09.MatrixOfNumbers/MatrixOfNumbers.cs
/*Problem 9. Matrix of Numbers

Write a program that reads from the console a positive integer number n (1 ≤ n ≤ 20) and prints a matrix.
*/
using System;
class MatrixOfNumbers
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine());
        int number = 1;

        for (int cols = 1; cols <= n; cols++)
        {
            number = cols;
            for (int rows = 1; rows <= n; rows++)
            {
                Console.Write("{0}", number + " ");
                number++;
            }
            Console.WriteLine();
        }

    }
}

File: JavaScript-Fundamentals/04.Loops/03. Matrix of Numbers/MatrixOfNumbers.js
/**Problem 4. Matrix of numbers
 * Write a javascript function that prints a matrix like in the examples below by a given integer N. Use two nested loops.
 Challenge: achieve the same effect without nested loops.
 3
 1 2 3
 2 3 4
 3 4 5
 */
function PrintMatrix(args) {
    var n = +args[0],
        row = '';

    for (var i = 1; i <= n; i++) {
        for (var j = i; j <= n + (i - 1); j++) {
            row += j + ' ';
        }
        console.log(row);
        row = '';
    }
}

// Check
PrintMatrix(['3']);

